src/vorta/views/repo_add_dialog.py

```diff
@@ -9,6 +10,8 @@
 from vorta.utils import borg_compat, choose_file_dialog, get_asset, get_private_keys, validate_passwords
 from vorta.views.utils import get_colored_icon
 
+logger = logging.getLogger(__name__)
```

Comment:
I added this because I wanted to know whether the function is getting triggered or not. If advised I'll remove it 😄